docs/references/glossary.md

```diff
@@ -0,0 +1,310 @@
+
+## Bundle
+
+A collection of [deployment artifacts](#deployment-artifacts).
+
+Can also be: The output of the bundling process, which brings a collection of [packages](#package) into a single collection of [deployment artifacts](#deployment-artifacts).
+
+## Catalog
+
+An organization's portfolio of software products managed in Backstage.
+
+## Cloud Native Computing
+
+A set of technologies that "empower organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach." ([CNCF Cloud Native Definition v1.0](https://github.com/cncf/toc/blob/main/DEFINITION.md)).
+
+## Cloud Native Computing Foundation
+
+A foundation dedicated to the promotion and advancement of [Cloud Native Computing](#Cloud-Native-Computing). The mission of the Cloud Native Computing Foundation (CNCF) is "to make cloud native computing ubiquitous" ([CNCF Charter](https://github.com/cncf/foundation/blob/main/charter.md)).
+
+CNCF is part of the [Linux Foundation](https://www.linuxfoundation.org/).
+
+## CNCF
+
+Cloud Native Computing Foundation.
+
+## Code Grant
+
+[OAuth](#oauth) flow where the client receives an [authorization code](#code) that is passed to the backend to be exchanged for an [access token](#access-token) and possibly a [refresh token](#refresh-token).
+
+## Collators
+
+Collators transform streams of [documents](#documents) into searchable texts. They're usually responsible for the data transformation and definition and collection process for specific [documents](#documents). Part of [Backstage Search](#search).
+
+## Component
+
+A software product that is managed in the Backstage [Software Catalog](#software-catalog). A component can be a service, website, library, data pipeline, or any other piece of software managed as a single project.
+
+https://backstage.io/docs/features/software-catalog/system-model
+
+## Condition
+
+Conditions are used to return a conditional decision from a policy. They contain information about a given entity and restrictions on what types of users can view that entity.
+
+## Conditional decisions
+
+[Rules](#permission-rules) need additional data before they can be used in a decision. Once a [rule](#permission-rule) is bound to relevant information it forms a [condition](#condition). Conditional decisions tell the [permission framework](#permission) to delegate evaluation to the [plugin](#plugin) that owns the corresponding [resource](#permission-resource). Permission requests that result in a conditional decision are allowed if all of the provided conditions evaluate to be true.
+
+## Contributor
+
+A volunteer who helps to improve an OSS product such as Backstage. This volunteer effort includes coding, testing, technical writing, user support, and other work. A [user role](#user-role).
+
+## Decorators
```

Comment:
Bit of a scoping / context problem here when reading it in isolation - could be `Search Decorator` or `Decorator (Search plugin)` or something like that? To immediately upfront know what we're talking about

Reply:
I tried Search decorators and didn't love it as it clumps all search terms next to each other, Decorators (search plugin) seems like a good alternative. The k8s glossary has an idea of tags that we could emulate as well, https://kubernetes.io/docs/reference/glossary/?all=true.